packages/blocks/header/package.json

```diff
@@ -69,6 +69,11 @@
   "blockprotocol": {
     "displayName": "Heading",
     "icon": "public/h1.svg",
-    "image": "public/preview.svg"
+    "image": "public/preview.svg",
+    "examples": [
+      {
+        "text": "Hello World!"
+      }
+    ]
```

Comment:
Can we give it an explicit `level: 1` and `color: "black"` prop too please? Makes it easier for people to see what they can change.

Reply:
Done